diff -Naur slim-1.3.5/slim.service slim-1.3.5.tpg/slim.service
--- slim-1.3.5/slim.service	2012-12-31 13:03:42.000000000 +0000
+++ slim-1.3.5.tpg/slim.service	2013-02-14 10:18:37.671715420 +0000
@@ -1,9 +1,14 @@
 [Unit]
 Description=SLiM Simple Login Manager
-After=systemd-user-sessions.service
+Documentation=man:slim(1)
+Conflicts=getty@tty7.service
+After=systemd-user-sessions.service getty@tty7.service plymouth-quit.service
 
 [Service]
 ExecStart=/usr/bin/slim -nodaemon
+StandardOutput=syslog
+Restart=always
+IgnoreSIGPIPE=no
 
 [Install]
 Alias=display-manager.service
